clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -1080,34 +1072,36 @@ protected void syncTopicSwitchToIngestionMetadataService(
       TopicSwitch topicSwitch,
       PartitionConsumptionState partitionConsumptionState,
       Map<String, Long> upstreamStartOffsetByKafkaURL) {
-    Optional<StoreVersionState> storeVersionState = storageMetadataService.getStoreVersionState(kafkaVersionTopic);
-    if (storeVersionState.isPresent()) {
-      String newTopicSwitchLogging = "TopicSwitch message (new source topic:" + topicSwitch.sourceTopicName
-          + "; rewind start time:" + topicSwitch.rewindStartTimestamp + "; upstream start offset by source Kafka URL: "
-          + upstreamStartOffsetByKafkaURL + ")";
-
-      if (storeVersionState.get().topicSwitch == null) {
-        LOGGER.info("First time receiving a " + newTopicSwitchLogging);
+    storageMetadataService.computeStoreVersionState(kafkaVersionTopic, previousStoreVersionState -> {
+      if (previousStoreVersionState != null) {
+        String newTopicSwitchLogging = "TopicSwitch message (new source topic:" + topicSwitch.sourceTopicName
+            + "; rewind start time:" + topicSwitch.rewindStartTimestamp
+            + "; upstream start offset by source Kafka URL: " + upstreamStartOffsetByKafkaURL + ")";
+
+        if (previousStoreVersionState.topicSwitch == null) {
+          LOGGER.info("First time receiving a " + newTopicSwitchLogging);
```

Comment:
lets follow the style mentioned in the style guide :) and on line 1084 as well. Thanks

Reply:
Sure, done for the whole file.